streamer/src/nonblocking/quic.rs

```diff
@@ -114,7 +114,7 @@ pub async fn run_server(
         Arc::new(Mutex::new(ConnectionTable::new(ConnectionPeerType::Staked)));
     while !exit.load(Ordering::Relaxed) {
         const WAIT_FOR_CONNECTION_TIMEOUT_MS: u64 = 1000;
-        const WAIT_BETWEEN_NEW_CONNECTIONS_US: u64 = 1000;
+        const WAIT_BETWEEN_NEW_CONNECTIONS_US: u64 = 5000;
```

Comment:
Why do we need to increase this sleep interval? It will reduce the timeliness of receiving the next connection.